modules/commandModule/commandModule.py

```diff
@@ -404,6 +465,248 @@ def set_begin_takeoff(self, beginTakeoff: bool):
+            return None
+        self.__pigoData.update({"initializingHomeBase": initializingHomeBase})
+        self.__write_to_pigo_file()
+
+    def set_flight_path_modify_next_id(self, flightPathModifyNextId: int):
+        """
+
+        Parameters
+        ----------
+        flightPathModifyNextId: int
+
+        """
+        if flightPathModifyNextId is None:
+            self.__logger.error("flightPathModifyNextId must be an int and not None.")
+            return None
+        if type(flightPathModifyNextId) is not int:
+            self.__logger.error("flightPathModifyNextId must be an int and not {}.".format(
+                type(flightPathModifyNextId)))
+            return None
+        self.__pigoData.update({"flightPathModifyNextId": flightPathModifyNextId})
+        self.__write_to_pigo_file()
+
+    def set_flight_path_modify_prev_id(self, flightPathModifyPrevId: int):
+        """
+
+        Parameters
+        ----------
+        flightPathModifyPrevId: int
+
+        """
+        if flightPathModifyPrevId is None:
+            self.__logger.error("flightPathModifyPrevId must be an int and not None.")
+            return None
+        if type(flightPathModifyPrevId) is not int:
+            self.__logger.error("flightPathModifyPrevId must be an int and not {}.".format(
+                type(flightPathModifyPrevId)))
+            return None
+        self.__pigoData.update({"flightPathModifyPrevId": flightPathModifyPrevId})
+        self.__write_to_pigo_file()
+
+    def set_flight_path_modify_id(self, flightPathModifyId: int):
+        """
+
+        Parameters
+        ----------
+        flightPathModifyId: int
+
+        """
+        if flightPathModifyId is None:
+            self.__logger.error("flightPathModifyId must be an int and not None.")
+            return None
+        if type(flightPathModifyId) is not int:
+            self.__logger.error("flightPathModifyId must be an int and not {}.".format(
+                type(flightPathModifyId)))
+            return None
+        self.__pigoData.update({"flightPathModifyId": flightPathModifyId})
+        self.__write_to_pigo_file()
+    
+    def set_holding_altitude(self, holdingAltitude: int):
+        """
+
+        Parameters
+        ----------
+        holdingAltitude: int
+
+        """
+        if holdingAltitude is None:
+            self.__logger.error("holdingAltitude must be an int and not None.")
+            return None
+        if type(holdingAltitude) is not int:
+            self.__logger.error("holdingAltitude must be an int and not {}.".format(
+                type(holdingAltitude)))
+            return None
+        self.__pigoData.update({"holdingAltitude": holdingAltitude})
+        self.__write_to_pigo_file()
+    
+    def set_holding_turn_radius(self, holdingTurnRadius: int):
+        """
+
+        Parameters
+        ----------
+        holdingTurnRadius: int
+
+        """
+        if holdingTurnRadius is None:
```

Comment:
Looking at all this repeated code makes me wonder if we can implement a builder pattern of sorts for this, again not for this PR but for future refactoring. As in have a class CommandBuilder(variableName) with methods buildTypeValidator(expectedType), and buildTypeValidator(...) returns a function called validate{variableName}(variable) that validates the type and makes the appropriate logs.